default.keys

```diff
@@ -1,3 +1,8 @@
+# This is a comment
+# Blank lines are also supported
+# You can put spaces between the colons
```

Comment:
I feel like this description doesn't need to be in this file. It's kinda obvious or we should document it elsewhere

Reply:
I plan on doing a follow up PR that uses something like TOML to do config, with something like a `[keys]` section, so this is more a stopgap.